### PR TITLE
Update group_1.yml

### PR DESCRIPTION
**Pull request by tbarry2**
Add my name to workflow file as comment

### CODE REVIEW

**Comment by tpugh** [comment]
@tbarry2 looks like your first pull request was with group 5. make this same update to [group_5.yml](https://github.com/tpugh/mutation-lab-class-2/blob/master/.github/workflows/group_5.ym) like you did here and your good to go.

File: .github/workflows/group_1.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: Group 1
-# Name 1, Name 2 Name 3, ...
+#Travis Barry
```
Comment (tpugh) [inline .github/workflows/group_1.yml:2]: @tbarry2 to match your reports file, make this edit on group_5.yml, you can do another pr and i can close this one.

**Review by tpugh — APPROVED** [review]

**Comment by tbarry2** [comment]
@tpugh the  annotation report creation failed with an unauthorized(403) error. Was this something I did wrong?

**Comment by tpugh** [comment]
@tbarry2 You did not do anything wrong, I put the action to trigger on pull_request and not just push, without giving you permissions so that was on my end. Here is that patch you made with everything working after I fixed my part. https://github.com/tpugh/mutation-lab-class-2/runs/6267905557?check_suite_focus=true

**Comment by tbarry2** [comment]
Thanks!

**Pull request merged**